lib/dynamoid/criteria.rb

```diff
@@ -13,12 +13,12 @@ module ClassMethods
         # see Dynamoid::Criteria::Chain.
         #
         # @since 0.2.0
-        define_method(meth) do |*args|
+        define_method(meth) do |*args, &blk|
           chain = Dynamoid::Criteria::Chain.new(self)
           if args
-            chain.send(meth, *args)
+            chain.send(meth, *args, &blk)
           else
-            chain.send(meth)
+            chain.send(meth, &blk)
```

Comment:
without this, the `Model.each` et al returned an enumerable even if it was passed a block.

Reply:
👍